### PR TITLE
Notes

### PR DESCRIPTION
**Pull request by rprzedzik**

### CODE REVIEW

File: module/comment/src/Application/Controller/Api/CommentChangeAction.php [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+     * @SWG\Tag(name="Comment")
+     * @SWG\Parameter(
+     *     name="language",
+     *     in="path",
+     *     type="string",
+     *     required=true,
+     *     default="EN",
+     *     description="Language Code",
+     * )
+     *
```
Comment (BastekBielawski) [inline module/comment/src/Application/Controller/Api/CommentChangeAction.php]: Unneccesery empty line

File: module/comment/src/Application/Controller/Api/CommentCreateAction.php [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\PropertyAccess\Exception\InvalidPropertyPathException;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/comments", methods={"POST"})
```
Comment (BastekBielawski) [inline module/comment/src/Application/Controller/Api/CommentCreateAction.php]: Name the route :)

File: module/comment/src/Application/Controller/Api/CommentGridAction.php [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+     */
+    public function __construct(CommentGrid $grid, CommentQueryInterface $query, GridRenderer $renderer)
+    {
+        $this->grid = $grid;
+        $this->query = $query;
+        $this->renderer = $renderer;
+    }
+
+    /**
+     * @IsGranted("PRODUCT_READ")
```
Comment (BastekBielawski) [inline module/comment/src/Application/Controller/Api/CommentGridAction.php]: This is ok? Product read?

File: module/comment/src/Application/Controller/Api/CommentChangeAction.php [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+     * @ParamConverter(class="Ergonode\Comment\Domain\Entity\Comment")
+     *
+     * @param Comment $comment
+     * @param Request $request
+     *
+     * @return Response
+     *
+     * @throws \Exception
+     */
+    public function __invoke(Comment $comment, Request $request): Response
```
Comment (BastekBielawski) [inline module/comment/src/Application/Controller/Api/CommentChangeAction.php]: Any one can change comments? No privilege?

File: module/comment/src/Application/Controller/Api/CommentCreateAction.php [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+     *     @SWG\Schema(ref="#/definitions/validation_error_response")
+     * )
+     *
+     * @param Request $request
+     *
+     * @return Response
+     *
+     * @throws \Exception
+     */
+    public function __invoke(Request $request): Response
```
Comment (BastekBielawski) [inline module/comment/src/Application/Controller/Api/CommentCreateAction.php]: No privilege?

File: module/comment/src/Application/Controller/Api/CommentDeleteAction.php [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+     *     description="Existing relationships"
+     * )
+     *
+     * @ParamConverter(class="Ergonode\Comment\Domain\Entity\Comment")
+     *
+     * @param Comment $comment
+     *
+     * @return Response
+     */
+    public function __invoke(Comment $comment): Response
```
Comment (BastekBielawski) [inline module/comment/src/Application/Controller/Api/CommentDeleteAction.php]: No privilege?

File: module/comment/src/Application/Controller/Api/CommentReadAction.php [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+     *     description="Not found",
+     * )
+     *
+     * @ParamConverter(class="Ergonode\Comment\Domain\Entity\Comment")
+     *
+     * @param Comment $comment
+     *
+     * @return Response
+     */
+    public function __invoke(Comment $comment): Response
```
Comment (BastekBielawski) [inline module/comment/src/Application/Controller/Api/CommentReadAction.php]: No privilege?

File: module/comment/src/Domain/Event/CommentContentChangedEvent.php [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+     * @JMS\Type("string")
+     */
+    private $to;
+
+    /**
+     * @var \DateTime
+     *
+     * @JMS\Type("DateTime")
+     */
+    private $editedAt;
```
Comment (BastekBielawski) [inline module/comment/src/Domain/Event/CommentContentChangedEvent.php]: Maybe updatedAt?

File: module/comment/src/Application/Controller/Api/CommentChangeAction.php (hunk 4 above)
Comment (rprzedzik) [reply module/comment/src/Application/Controller/Api/CommentChangeAction.php]: No

File: module/comment/src/Application/Controller/Api/CommentCreateAction.php (hunk 5 above)
Comment (rprzedzik) [reply module/comment/src/Application/Controller/Api/CommentCreateAction.php]: No

File: module/comment/src/Application/Controller/Api/CommentDeleteAction.php (hunk 6 above)
Comment (rprzedzik) [reply module/comment/src/Application/Controller/Api/CommentDeleteAction.php]: No

File: module/comment/src/Domain/Event/CommentContentChangedEvent.php (hunk 8 above)
Comment (rprzedzik) [reply module/comment/src/Domain/Event/CommentContentChangedEvent.php]: User edit the note...

**Pull request merged**